PF/Assignment2/PascalTringle.java

```diff
@@ -0,0 +1,68 @@
+public class PascalTringle {
+
+	public static String spaces(int row, int n) {
+
+		if (row < 2 * n && row > 0) {
+			String spaceOutput = "";
+			int space;
+			if (row > n) {
+				space = row - n;
+			} else {
+				space = n - row;
+			}
+
+			for (int count = 0; count < space; count++) {
+
+				spaceOutput = spaceOutput.concat(" ");
+			}
+
+			return spaceOutput;
+		} else {
+			return "wrong Input";
```

Comment:
You should never return anything which is not asked. This method should only return "spaces", since you are dealing with some unwanted scenario (in this case it is row > 2*n) you should throw exception e.g. 
      throw new Exception("wrong Input");